.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**.yaml"
+      - "**.yml"
+      - "**.tpl"
+      # Exclude build index.yaml from being collected into action
```

Comment:
This exclusion looks redundant, since `GITHUB_TOKEN` does not recursively call the same action. More information needed.